README.md

```diff
@@ -150,7 +150,48 @@ target:
 `subpath` in the previous url is optional in case your charts are not stored directly under your projects.
 It is worth mentioning that you can use Harbor robot accounts using OCI registries as source or target.
 
-Also, take into account that if you use OCI as the source repository you must specify the list of chart to synchronize
+Also, take into account that if you use OCI as the source repository you must specify the list of charts to synchronize
+or a pointer to an [index file](#oci-index) in the repository.
+
+#### OCI index
+
+In order to speed up the sync process for OCI charts, it is possible to specify a reference to an OCI index file containing 
```

Comment:
To me, the main value of this feature is not speeding up the process but auto-discover what charts to synchronize without maintaining a hardcoded list.

The speed side-effect is a nice one but it's just an optimization.